internal/rust/template.Dockerfile

```
@@ -42,23 +54,14 @@ RUN apt-get update \
 {{ end }}
 
 RUN useradd -m -s /bin/bash zeabur
-COPY --from=builder --chown=zeabur:zeabur /app /app
 
 USER zeabur
 WORKDIR /app
 
-ENV BINDIR="/app/bin"
-ENV BINNAME="{{ .BinName }}"
-ENV EXEFILE="${BINDIR}/${BINNAME}"
-
-RUN if [ ! -x "${EXEFILE}" ]; then \
-    find . -type f -executable -print | head -n 1 > EXEFILE; \
-  else \
-    echo "${EXEFILE}" > EXEFILE; \
-  fi
+COPY --from=builder --chown=zeabur:zeabur /app/bin/server ./bin/server
```

```suggestion
COPY --from=builder --chown=zeabur:zeabur /app/bin/server /app/bin/server
```

Better to be explicit.